<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsPage</class>
 <widget class="QWidget" name="SettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tabUserInterface">
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="horizontalSpacing">
        <number>8</number>
       </property>
       <property name="verticalSpacing">
        <number>8</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblTheme"/>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cmbTheme"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblUpdates"/>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="chkUpdates"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
